content/QRadar-SIEM/104.es.md

```diff
@@ -0,0 +1,175 @@
+
+## Visualización de fuentes de registro autodescubiertas
+
+QRadar autodescubre muchas fuentes de registro después de recibir varios registros de un tipo específico. No necesita configurar la fuente de registro para que QRadar reconozca los eventos de esa fuente de registro. En este ejercicio, analizará una fuente de registro autodescubierta.
+
+1.  Empiece por la Consola. Para cargar los filtros de búsqueda predeterminados, haga doble clic en la pestaña **Actividad de registro**.
+
+2.  A continuación, en el terminal SSH de QRadar, ejecute el siguiente script desde el directorio /labfiles/extended/labfiles/logsources:
+
+    ```bash
+    /opt/qradar/bin/logrun.pl -f checkpoint.syslog 10
+    ```
+
+3.  Vuelva a la ventana del navegador y observe los eventos que se muestran en la Consola.
+
+4.  Transcurridos unos 30 segundos, observará que la columna Event Name comienza a mostrar Firewall Permit, el LogSource se denomina Check Point y Low Level Category se actualiza a Firewall Permit.
+
+    ![](images/104/image8.jpeg)
+
+5.  Para pausar los eventos en tiempo real, en la parte superior derecha de la Consola, haga clic en el icono de **pausa**.
+
+6.  Haga doble clic en cualquier evento **Firewall Permit** y observe que la IP de origen y la IP de destino analizadas coinciden con las direcciones IP de la carga útil sin procesar.
+
+    ![](images/104/image9.jpeg)
+
+    Usted aprendió que QRadar puede autodescubrir algunos registros después de recibir algunos eventos. Sin embargo, no todos los registros son autodescubiertos, y para estos, usted debe crear fuentes de registro manualmente usando la aplicación Log Source Management. Si no quiere perder parte de la información de los primeros eventos mientras espera a que QRadar los analice, también puede utilizar este método para las fuentes de registro autodescubiertas.
+
+## Utilice la aplicación Log Source Management para configurar una fuente de registro
+
+En este ejercicio, configurará un origen de registro de Symantec Endpoint Protection y analizará los eventos de este origen de registro. Para configurar un origen de registro, debe utilizar la aplicación Log Source Management (LSM).
+
+1.  Desde el navegador, vaya a la Admin Console.
+
+2.  En Admin Console, desplácese hasta la sección Apps y haga clic en **QRadar Log Source Management**.
+
+3.  En la aplicación LSM, haga clic en **Fuentes de registro.**
+
+4.  En el panel de control de la aplicación LSM, en la parte superior derecha de la ventana, haga clic en **Nueva fuente de registro**.
+
+    ![](images/104/image12.png)
+
+5.  Haga clic en **Fuente de registro única**.
+
+    ![](images/104/image13.jpeg)
+
+    > Deténgase y navegue a través de la larga lista de fuentes de registro disponibles. Esto le muestra todas las fuentes de registro con las que QRadar puede integrarse.
```

Comment:
There was a typo "browser" instead of "browse" in the original english version. The translated version with the typo fixed matches this

Reply:
e.g. no changes needed.